src/data-requests/r-value.ts

```diff
@@ -9,7 +9,7 @@ function parseRValue(data: ArrayBuffer): { r: number, date: Date } | null {
 
     const latestEntry = json[json.length - 1];
     const dateString = latestEntry["Datum des Erkrankungsbeginns"];
-    let rValue = latestEntry["Punktschätzer des 4-Tage-R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert"];
+    let rValue = latestEntry["Punktschätzer des 4-Tage R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert" || latestEntry["Punktschätzer des 4-Tage-R-Wertes"];
 
```

Comment:
ups, there is a ] missing!
let rValue = latestEntry["Punktschätzer des 4-Tage R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert"] || latestEntry["Punktschätzer des 4-Tage-R-Wertes"];

Reply:
sorry, I overlooked that.

Reply:
I suggested that also wrong! so thats my misstake!